coe/data-analytics.html

```diff
@@ -1,91 +1,40 @@
 ---
-layout: default
+layout: coe
+title: Data Analytics
+tagline: Helps make more efficient use of data management, analysis, and reporting capabilities.
+page-nav:
+  children:
+    - agency: HUD
+      url-fragment: hud.html#data%20analytics
+    - agency: USDA
+      url-fragment: usda.html#data%20analytics
+intro: The Data Analytics CoE’s primary focus is to assist agencies increasing the frequency and quality of data-driven, evidence based decisions at both the executive and program delivery level with a particular focus on incorporating customer experience data in driving process improvements. The team helps drive innovation at partner agencies in data management, organizational capacity, analysis and enterprise analytics infrastructure.
+service-offerings:
+  - Data analysis and reporting
+  - Interactive dashboard creation
+  - Continuous process improvement
+  - Metric definition and measurement
+  - Data driven process implementation
+  - Embedding mechanisms for rapid feedback and analysis
+service-catalog:
+  children:
+    - description: |
+        "The Data Analytics CoE helps make more efficient use of data management, analysis, and reporting capabilities.
+        "To be successful, the Data Analytics CoE’s services employ an approach that simultaneously identifies incremental improvements to the present state while envisioning a desired future state that will achieve greater advancements in program outcomes. The team’s methodology is based on an open-source framework for mature information management that addresses a range of core competencies, including:"
+      list:
+        - business process re-engineering,
+        - business intelligence,
+        - enterprise data collection and management,
+        - data governance, and
+        - data visualization and reporting.
+sidebar-graphic-class: coe-da
+pdfs:
+  - title: Data Analytics Service Catalog
+    link: /docs/Data_Analytics__One_Page_Service_Catalog.pdf
+sidebar-extra-content:
+  children:
```

Comment:
Note that the `children` keyword here sets up this extra content in a way that is extensible -- you can potentially have multiple boxes in the sidebar if you like.